tools/extract-release.sh

```diff
@@ -0,0 +1,10 @@
+gateway_old=./gateway
+$gateway_old/tools/extract-ca-archive gateway-*.tar.gz gateway_new
+pushd gateway_new
+$gateway_old/tools/extract-ca-archive ../node_modules-*.tar.gz node_modules
+cp -r $gateway_old/db.sqlite3 $gateway_old/settings ./
+popd
+systemctl gateway.service stop
```

Comment:
Shouldn't this also be copying the *.pem files and the tunneltoken file?

Reply:
I changed the approach used in `tools/upgrade.sh` where instead of copying single files it copies the entire `gateway` directory then uses tar to overwrite the files. I also moved the `systemctl gateway.service stop` to much earlier in the process so there will be enough time for the gateway to shut down cleanly.

Reply:
Perhaps just add a comment then where that copy occurs that it will copy the pem files, tunneltoken, and db. That way somebody searching will find it and it will act as a reminder in case somebody tries to change it.